pysyncrosim/environment.py

```
@@ -178,33 +181,60 @@ def update_run_log(*message, sep="", type="status"):
+    """
     e = _environment()
-    
-    if e.program_directory.item() is None:
-        raise RuntimeError("This function requires a SyncroSim environment")
+
+    if require_vars is None:
+        # If no specific vars required, just check that ANY env var is set
+        # This indicates we're running inside SyncroSim
+        if all(e[col].item() is None or (isinstance(e[col].item(), int) and e[col].item() == -1)
+               for col in e.columns):
+            raise RuntimeError("This function requires a SyncroSim environment")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)